Rebase to 1.8.7

Cardinal doesn't work on SportBukkit 1.8.7

@dentmaged That isn't what is causing this issue. The problem is Cardinal hard codes NBT rather than using reflection, it appears that some of these paths have changed in the SportBukkit 1.8.7 rebase. There are several of these errors through out the program, however you can reflect on this code snippet as a brief example.

```
[10:53:37] [Server thread/WARN]: java.lang.NoClassDefFoundError: net/minecraft/server/v1_8_R1/Packet
[10:53:37] [Server thread/WARN]: 	at in.twizmwaz.cardinal.module.modules.bossBar.BossBarBuilder.load(BossBarBuilder.java:10)
[10:53:37] [Server thread/WARN]: 	at in.twizmwaz.cardinal.module.ModuleFactory.build(ModuleFactory.java:199)
[10:53:37] [Server thread/WARN]: 	at in.twizmwaz.cardinal.match.Match.registerModules(Match.java:49)
[10:53:37] [Server thread/WARN]: 	at in.twizmwaz.cardinal.GameHandler.cycleAndMakeMatch(GameHandler.java:52)
[10:53:37] [Server thread/WARN]: 	at in.twizmwaz.cardinal.GameHandler$1.run(GameHandler.java:36)
[10:53:37] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftTask.run(CraftTask.java:53)
[10:53:37] [Server thread/WARN]: 	at org.bukkit.craftbukkit.v1_8_R3.scheduler.CraftScheduler.mainThreadHeartbeat(CraftScheduler.java:349)
[10:53:37] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.MinecraftServer.B(MinecraftServer.java:696)
[10:53:37] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.DedicatedServer.B(DedicatedServer.java:335)
[10:53:37] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.MinecraftServer.A(MinecraftServer.java:645)
[10:53:37] [Server thread/WARN]: 	at net.minecraft.server.v1_8_R3.MinecraftServer.run(MinecraftServer.java:555)
[10:53:37] [Server thread/WARN]: 	at java.lang.Thread.run(Unknown Source)
[10:53:37] [Server thread/WARN]: Caused by: java.lang.ClassNotFoundException: net.minecraft.server.v1_8_R1.Packet
[10:53:37] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:72)
[10:53:37] [Server thread/WARN]: 	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:67)
[10:53:37] [Server thread/WARN]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[10:53:37] [Server thread/WARN]: 	at java.lang.ClassLoader.loadClass(Unknown Source)
[10:53:37] [Server thread/WARN]: 	... 12 more
```

--- a/src/main/java/in/twizmwaz/cardinal/module/modules/playable/Playable.java
+++ b/src/main/java/in/twizmwaz/cardinal/module/modules/playable/Playable.java
@@ -29,7 +29,7 @@ public class Playable implements Module {
 
     @EventHandler
     public void onPlayerMove(PlayerMoveEvent event) {
-        if (GameHandler.getGameHandler().getMatch().isRunning() && !TeamUtils.getTeamByPlayer(event.getPlayer()).isObserver()) {
+        if (GameHandler.getGameHandler().getMatch().isRunning() && TeamUtils.getTeamByPlayer(event.getPlayer()) != null && !TeamUtils.getTeamByPlayer(event.getPlayer()).isObserver()) {
             if (region != null) {
                 if (region.contains(event.getTo().toVector()) && !region.contains(event.getFrom().toVector())) {
                     ChatUtils.sendWarningMessage(event.getPlayer(), new LocalizedChatMessage(ChatConstant.ERROR_PLAYABLE_LEAVE));
@@ -41,7 +41,7 @@ public class Playable implements Module {
 
     @EventHandler
     public void onBlockPlace(BlockPlaceEvent event) {
-        if (GameHandler.getGameHandler().getMatch().isRunning() && !TeamUtils.getTeamByPlayer(event.getPlayer()).isObserver()) {
+        if (GameHandler.getGameHandler().getMatch().isRunning() && TeamUtils.getTeamByPlayer(event.getPlayer()) != null && !TeamUtils.getTeamByPlayer(event.getPlayer()).isObserver()) {
             if (region != null) {
                 if (region.contains(event.getBlock().getLocation())) {
                     ChatUtils.sendWarningMessage(event.getPlayer(), new LocalizedChatMessage(ChatConstant.ERROR_PLAYABLE_INTERACT));
@@ -53,7 +53,7 @@ public class Playable implements Module {
 
     @EventHandler
     public void onBlockBreak(BlockBreakEvent event) {
-        if (GameHandler.getGameHandler().getMatch().isRunning() && !TeamUtils.getTeamByPlayer(event.getPlayer()).isObserver()) {
+        if (GameHandler.getGameHandler().getMatch().isRunning() && TeamUtils.getTeamByPlayer(event.getPlayer()) != null && !TeamUtils.getTeamByPlayer(event.getPlayer()).isObserver()) {
             if (region != null) {
                 if (region.contains(event.getBlock().getLocation())) {
                     ChatUtils.sendWarningMessage(event.getPlayer(), new LocalizedChatMessage(ChatConstant.ERROR_PLAYABLE_INTERACT));
@@ -65,7 +65,7 @@ public class Playable implements Module {
 
     @EventHandler
     public void onPlayerInteract(PlayerInteractEvent event) {
-        if (GameHandler.getGameHandler().getMatch().isRunning() && !TeamUtils.getTeamByPlayer(event.getPlayer()).isObserver()) {
+        if (GameHandler.getGameHandler().getMatch().isRunning() && TeamUtils.getTeamByPlayer(event.getPlayer()) != null && !TeamUtils.getTeamByPlayer(event.getPlayer()).isObserver()) {
             if (region != null && event.getClickedBlock() != null) {
                 if (region.contains(event.getClickedBlock().getLocation())) {
                     ChatUtils.sendWarningMessage(event.getPlayer(), new LocalizedChatMessage(ChatConstant.ERROR_PLAYABLE_INTERACT));

